reminders/script.js

```diff
@@ -1,6 +1,70 @@
+var toDoList = getToDos() || [];
+
+
 function createItem() {
-    var list = document.getElementById("list-items").value;
-    var todo = document.getElementById("todo");
-    todo.insertAdjacentHTML('beforeend', '<input type="checkbox"> ' + list + '<br>');
-    console.log(list)
-}
\ No newline at end of file
+    var newToDo = addNewTodo();
+    updateTodoList(newToDo);
+    saveTodosLocally();
+    addToDoToHTML(newToDo);
+    console.log(toDoList);
+   }
+
+class Todo {
+    constructor (text, completed, id) {
+        this.text = text;
+        this.completed = completed || false;
+        this.id = id || Math.random().toString(36).substring(7);
+    }
+
+}
+
+
+function checkIfCompleted(btn) {
```

Comment:
this is probably the only function in this file that seems too complicated and bloated to me. One of the main reasons I decided to go the route of using a class for a todo item is b/c it will let us act on a single todo items, e.g. `var todoItem = new Todo(...)` creates a todo for us but that instance can have additional functionality in it, such as `todoItem.markAsDone()` or `todoItem.markAsIncomplete()` that way you wouldn't have to do the following function.